idPhyRufi1_test.yaml

```diff
@@ -0,0 +1,20 @@
+samples:
+    - id: idPhyRufi1
+      assembly:
+        primary: /lustre/scratch123/tol/projects/darwin/data/insects/Physocephala_rufipes/working/idPhyRufi1.hifiasm.20220613/purging/purged.fa.gz 
+        haplotigs: /lustre/scratch123/tol/projects/darwin/data/insects/Physocephala_rufipes/working/idPhyRufi1.hifiasm.20220613/purging/purged.htigs.fa.gz
+      illumina:
+        reads: /lustre/scratch123/tol/projects/darwin/data/insects/Physocephala_rufipes/genomic_data/idPhyRufi1/10x/
+        kmer_pref: /lustre/scratch123/tol/projects/darwin/data/insects/Physocephala_rufipes/genomic_data/idPhyRufi1/10x/kmer/k31/idPhyRufi1.k31
+      pacbio:
+        reads: 
+            - reads: /lustre/scratch123/tol/projects/darwin/data/insects/Physocephala_rufipes/genomic_data/idPhyRufi1/pacbio/fasta/m64094e_210524_093354.ccs.bc1019_BAK8B_OA--bc1019_BAK8B_OA.filtered.fasta.gz
+        kmer_pref: /lustre/scratch123/tol/projects/darwin/data/insects/Physocephala_rufipes/genomic_data/idPhyRufi1/pacbio/kmer/k31/idPhyRufi1.k31
+      HiC:
```

Comment:
Shouldn't this include 10x as well?

Reply:
I think 10x should be labelled as such. But do you have regular Illumina data?

Reply:
Haven't seen any other Illumina data rather than 10X in production